frontend/pages/howToPlay.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="../styles/HowToPlay.css">
+	<script type="module" src="../scripts/index.js"></script> 
+	<title>How To Play</title>
+</head>
+<body style="background-color: lightyellow;">
+    <div id="howToPlayContainer">
+        <h1 id="text"> How To Play: </h1>
+        <canvas id="howtoplay">How To Play</canvas>
+        
+        
+
+    </div>
+
+    <br />
+    <body> The aim in Tetris is simple; 
```

Comment:
Using another `<body>` tag is quite strange, can you replace this with a `<div>` instead?